2019-12-12 10:19:40,886 INFO 6519 [egg-logrotator] agent logger reload: got log-reload message
2019-12-12 17:51:03,144 INFO 965 [egg:logger] init all loggers with options: {"dir":"/Users/alanj/Desktop/myprojects/egg-demo/logs/egg-demo","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"egg-demo-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2019-12-12 17:51:03,155 INFO 965 [egg:core] dump config after load, 4ms
2019-12-12 17:51:03,194 INFO 965 [egg-watcher] Start watching: ["/Users/alanj/Desktop/myprojects/egg-demo/app","/Users/alanj/Desktop/myprojects/egg-demo/config","/Users/alanj/Desktop/myprojects/egg-demo/mocks","/Users/alanj/Desktop/myprojects/egg-demo/mocks_proxy","/Users/alanj/Desktop/myprojects/egg-demo/app.js"]
2019-12-12 17:51:03,194 INFO 965 [egg-watcher] Start watching: "/Users/alanj/Desktop/myprojects/egg-demo/app"
2019-12-12 17:51:03,195 INFO 965 [egg-watcher] Start watching: "/Users/alanj/Desktop/myprojects/egg-demo/config"
2019-12-12 17:51:03,195 INFO 965 [egg-watcher] Start watching: "/Users/alanj/Desktop/myprojects/egg-demo/mocks"
2019-12-12 17:51:03,195 INFO 965 [egg-watcher] Start watching: "/Users/alanj/Desktop/myprojects/egg-demo/mocks_proxy"
2019-12-12 17:51:03,195 INFO 965 [egg-watcher] Start watching: "/Users/alanj/Desktop/myprojects/egg-demo/app.js"
2019-12-12 17:51:03,195 INFO 965 [egg-watcher:agent] watcher start success
2019-12-12 17:51:03,205 INFO 965 [egg:core] dump config after ready, 3ms
2019-12-12 19:54:30,528 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 165237ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 19:57:09,931 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 139397ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:01:15,793 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 245858ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:06:29,597 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 293782ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:21:57,147 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 927544ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:37:06,428 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 909280ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:56:02,920 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 1136488ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:57:35,409 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 72477ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 20:58:41,553 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 66139ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 21:32:04,083 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 1982526ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 23:37:55,719 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 1290638ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-12 23:53:39,263 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 943529ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 01:31:22,255 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 3718090ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 03:25:02,729 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 6820270ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 03:30:38,772 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 336007ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 05:18:49,731 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 6490932ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 05:29:48,672 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 604426ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 05:41:54,538 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 725822ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 05:52:19,036 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 604465ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:02:37,028 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 597975ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:12:45,563 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 608478ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:23:10,790 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605197ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:33:37,315 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 606507ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:43:42,720 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605383ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 06:54:06,645 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 603892ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 07:04:31,847 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605182ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 07:25:39,597 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 1267585ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 07:36:03,301 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 603672ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 07:49:29,959 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 786623ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 07:59:35,952 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605977ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:10:00,158 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 604178ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:20:05,574 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605318ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:30:30,379 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 604777ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:40:54,958 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 604556ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:45:05,021 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 250029ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:51:05,411 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 360377ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 08:52:11,487 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 66072ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:13:42,963 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 1271460ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:23:48,715 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 605749ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:28:17,839 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 249099ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:39:35,441 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 637527ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:55:39,106 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 943644ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 09:58:31,950 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 132267ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

2019-12-13 10:00:21,724 ERROR 965 nodejs.ClusterClientNoResponseError: client no response in 69754ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/alanj/Desktop/myprojects/egg-demo/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 965
hostname: localhost

